### PR TITLE
Creating modals

### PR DESCRIPTION
**Pull request by dpodsobilov**
PlaceType, cinemahall modals were been created

### CODE REVIEW

File: client/src/app/services/admin/admin-cinemas.service.ts [hunk 1]
```diff
@@ -42,6 +48,19 @@ export class AdminCinemasService {
       },
     );
   }
+
+  addHall(hall: AdminHallCreation) {
```
Comment (dpodsobilov) [inline client/src/app/services/admin/admin-cinemas.service.ts:52]: Request for adding cinemahall to the server

File: client/src/app/services/admin/admin-cinemas.service.ts [hunk 2]
```diff
@@ -42,6 +48,19 @@ export class AdminCinemasService {
...
+    return this.http.post(this.baseUrl + '/Admin/Hall?', hall, {
+      observe: 'response',
+    });
+  }
+
+  editHall(hall: AdminHall) {
```
Comment (dpodsobilov) [inline client/src/app/services/admin/admin-cinemas.service.ts:58]: Request for editing cinemahall to the server

File: client/src/app/services/admin/admin-cinemas.service.ts [hunk 3]
```diff
@@ -1,36 +1,42 @@
...
 import { Observable } from 'rxjs';
-export interface AdminCinemas {
+export interface AdminCinema {
   cinemaId: number;
   cinemaName: string;
   cinemaAddress: string;
 }
 
-export interface AdminHalls {
+export interface AdminHall {
```
Comment (dpodsobilov) [inline client/src/app/services/admin/admin-cinemas.service.ts:10]: Interface for editing cinemahall

File: client/src/app/services/admin/admin-cinemas.service.ts [hunk 4]
```diff
@@ -1,36 +1,42 @@
...
   cinemaHallId: number;
   cinemaHallName: string;
   cinemaHallTypeName: string;
   cinemaId: number;
 }
 
+export interface AdminHallCreation {
```
Comment (dpodsobilov) [inline client/src/app/services/admin/admin-cinemas.service.ts:17]: Interface for creating cinemahall

File: client/src/app/services/admin/place-type.service.ts [hunk 5]
```diff
@@ -2,13 +2,21 @@ import { Inject, Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import { Observable } from 'rxjs';
 import { AdminTemplates } from './admin-template.service';
+import { AdminGenre, AdminGenreCreation } from './admin-film.service';
 
 export interface PlaceType {
```
Comment (dpodsobilov) [inline client/src/app/services/admin/place-type.service.ts:7]: Interface for editingPlaceType

File: client/src/app/services/admin/place-type.service.ts [hunk 6]
```diff
@@ -2,13 +2,21 @@ import { Inject, Injectable } from '@angular/core';
...
   typeId: number;
   name: string;
   color: string;
   cost: number;
 }
+
+export interface AdminPlaceTypeCreation {
```
Comment (dpodsobilov) [inline client/src/app/services/admin/place-type.service.ts:14]: Interface for creating PlaceType

File: client/src/app/services/admin/place-type.service.ts [hunk 7]
```diff
@@ -21,4 +29,25 @@ export class PlaceTypeService {
   getTypes(): Observable<PlaceType[]> {
     return this.http.get<PlaceType[]>(this.baseUrl + '/Admin/PlaceType');
   }
+
+  addType(placeType: AdminPlaceTypeCreation) {
```
Comment (dpodsobilov) [inline client/src/app/services/admin/place-type.service.ts:33]: Request for adding placeType to the server

File: client/src/app/services/admin/place-type.service.ts [hunk 8]
```diff
@@ -21,4 +29,25 @@ export class PlaceTypeService {
...
+    return this.http.post(this.baseUrl + '/Admin/PlaceType?', placeType, {
+      observe: 'response',
+    });
+  }
+
+  editType(placeType: PlaceType) {
```
Comment (dpodsobilov) [inline client/src/app/services/admin/place-type.service.ts:39]: Request for editing placeType to the server

File: client/src/app/services/admin/place-type.service.ts [hunk 9]
```diff
@@ -21,4 +29,25 @@ export class PlaceTypeService {
...
+    return this.http.put(this.baseUrl + '/Admin/PlaceType?', placeType, {
+      observe: 'response',
+    });
+  }
+
+  deleteType(placeTypeId: number) {
```
Comment (dpodsobilov) [inline client/src/app/services/admin/place-type.service.ts:45]: Request for deleting placeType to the server

File: server/Logic/Commands/Admin/EditCommand/EditTemplateCommand.cs [hunk 10]
```diff
@@ -72,6 +72,10 @@ public async Task Handle(EditTemplateCommand request, CancellationToken cancella
         
         // Само редактирование
         cinemaHallType.CinemaHallTypeName = request.CinemaHallTypeName;
+        var oldPlaces = await _applicationContext.Places
```
Comment (dpodsobilov) [inline server/Logic/Commands/Admin/EditCommand/EditTemplateCommand.cs:75]: old uncommited command on backend

**Review by dpodsobilov — COMMENTED** [review]
Infromation for backend

**Review by Ruslanilin26 — APPROVED** [review]

**Pull request merged**